sambaplayersdk/src/main/java/com/sambatech/player/SambaPlayer.java

```diff
@@ -374,7 +376,15 @@ public void onConnected(final CastSession castSession) {
             sambaCast.registerDeviceForProgress(true);
             castPlayer.setMessageListener(castSession);
 
-            simplePlayerView.showCast();
+            simplePlayerView.showCast(media.isLive, new CastLiveButtonListener() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this anonymous inner class a lambda (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.sambatech.com/coding_rules#rule_key=squid%3AS1604)